.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: "node_js"
```

Comment:
with this travis.yml you're only testing your front-end code, not your back-end code